wemake_python_styleguide/violations/best_practices.py

```diff
@@ -2812,6 +2812,7 @@ def func(name):
         :str:`wemake_python_styleguide.options.defaults.EXPS_FOR_ONE_EMPTY_LINE`
 
     .. versionadded:: 0.17.0
+    .. versionchanged:: 0.18.0
```

Comment:
No need to specify this, it is for major changes, not bugfixes.

```suggestion
    .. versionchanged:: 0.18.0
```

Reply:
Okey, I will remove this line. I can't commit suggestion